internal/provider/provider_test_utils.go

```diff
@@ -47,11 +47,14 @@ func TestAccPreCheck(t *testing.T) {
 	}
 }
 
-func ProviderConfig(t *testing.T, isHosted bool) string {
+func ProviderConfig(t *testing.T, isHosted bool, isScim bool) string {
```

Comment:
added this param to the signature and had to update all the files accordingly (which is why all the test files were affected)